tests/api.rs

```diff
@@ -202,6 +216,79 @@ fn text_pos_03() {
     assert_eq!(doc.text_pos_at(node.range().end), TextPos::new(2, 5));
 }
 
+#[cfg(feature = "positions-extra-attr")]
+#[test]
+fn text_pos_04() {
+    let data = "<n1:e xmlns:n1='http://www.w3.org' n1:a=''/>";
+
+    let doc = Document::parse(data).unwrap();
+    let node = doc.root_element();
+
+    if let Some(attr) = node.attribute_node("a") {
+        assert_eq!(doc.text_pos_at(attr.range().start), TextPos::new(1, 36));
+        assert_eq!(doc.text_pos_at(attr.range().end), TextPos::new(1, 43));
+        assert_eq!(doc.text_pos_at(attr.range_qname().start), TextPos::new(1, 36));
+        assert_eq!(doc.text_pos_at(attr.range_qname().end), TextPos::new(1, 40));
+        assert_eq!(doc.text_pos_at(attr.range_value().start), TextPos::new(1, 42));
+        assert_eq!(doc.text_pos_at(attr.range_value().end), TextPos::new(1, 42));
+}
+}
+
+#[cfg(feature = "positions-extra-attr")]
+#[test]
+fn text_pos_05() {
+    let data = "<n1:e xmlns:n1='http://www.w3.org' n1:a  =   'b'/>";
+
+    let doc = Document::parse(data).unwrap();
+    let node = doc.root_element();
+
+    if let Some(attr) = node.attribute_node("a") {
+        assert_eq!(doc.text_pos_at(attr.range().start), TextPos::new(1, 36));
+        assert_eq!(doc.text_pos_at(attr.range().end), TextPos::new(1, 48));
+        assert_eq!(doc.text_pos_at(attr.range_qname().start), TextPos::new(1, 36));
+        assert_eq!(doc.text_pos_at(attr.range_qname().end), TextPos::new(1, 40));
+        assert_eq!(doc.text_pos_at(attr.range_value().start), TextPos::new(1, 47));
+        assert_eq!(doc.text_pos_at(attr.range_value().end), TextPos::new(1, 48));
+    }
+}
+
+#[cfg(feature = "positions-extra-attr")]
```

Comment:
It's fine to skip those this test. u16 is plenty enough.

Reply:
Wasn't sure if you meant to remove both tests, or only the one for qname exceeding the limit (I agree that one's unnecessary if the limit is `u16`).  For now I left the test for `eq_len` maxing out, but I can remove that one too if you want.